### PR TITLE
feat:remove liquidity proposal

### LINKED ISSUE #73: UI to remove liquidity from pair

### PR DESCRIPTION
**Pull request by Macxim**
Fix #73

### CODE REVIEW

**Comment by philipdesmedt** [comment]
@Macxim 

Screenshot for reference
image

Can we add the current LP token balance somewhere, so they know how much they can remove if they fill in the input?

**Comment by Macxim** [comment]
Right. Good point. I'll add it. Thanks for the screenshot 😅

**Comment by philipdesmedt** [comment]
@Macxim this is done.. let me know if you have any comments or questions

File: web/components/remove-swap-liquidity.tsx [hunk 1]
```diff
@@ -54,180 +58,294 @@ export const RemoveSwapLiquidity: React.FC = ({ match }) => {
...
+                        1 {tokenX.name} = {tokenXPrice} {tokenY.name}
+                      </p>
+                      <p  >
+                        1 {tokenY.name} = {tokenYPrice} {tokenX.name}
+                      </p>                    
+                    </div>
+                  </div>
+                  
+                  <div className="mt-4 lg:flex lg:items-center lg:flex-1 lg:space-x-2">
+                    <button
```
Comment (Macxim) [inline web/components/remove-swap-liquidity.tsx:333]: We probably need some feedback/message/disabled state like we did for other buttons (see Swap and Add liquidity pages)

**Pull request merged**